GithubJobApp/UINetImageView.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  UINetImageView.swift
+//  GithubJobApp
+//
+//  Created by Daniel Harrison on 12/1/16.
+//  Copyright © 2016 Taylor Harrison. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class UINetImageView: UIImageView {
+    
+    static let netImageCache = NSCache()
+    
+    func setImgFromUrl(url: String) {
+        if let imgUrl = NSURL(string: url) {
+            let task = NSURLSession.sharedSession().dataTaskWithURL(imgUrl) {(data, response, error) in
+                if error == nil {
+                    dispatch_async(dispatch_get_main_queue()) { () -> Void in
+                        if let data = data, logoImage = UIImage(data: data) {
```

Comment:
Minor detail -- you'll usually want to create the image (`UIImage(data: data)`) on the background thread and then load the image on the main thread.  For larger images, the time to decompress the image can be non-trivial, so you don't want to tie up the main thread with that extra processing.